detekt.yml

```diff
@@ -0,0 +1,15 @@
+comments:
+  UndocumentedPublicClass:
+    active: true
+    includes: "**/commonMain/**"
+  UndocumentedPublicProperty:
+    active: true
+    includes: "**/commonMain/**"
```

Comment:
I think it makes sense to force documentation of public facing components at least in commonMain since this will be what users will work with